vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_24.py

```diff
@@ -112,6 +156,18 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
         :param layer: The layer with the weights to be processed
         
         """
+        if self.do_sparse_decompress:
+            layer.weight.data = self._decompress_bitmask_compressed_weight(
+                compressed=layer.compressed,
+                bitmask=layer.bitmask,
+                layer=layer,
+            )
+            # compressed and bitmask tensors
+            # are no longer needed after decompression
+            
+            del layer.compressed
+            del layer.bitmask
```

Comment:
nit: better whitespace like this?
```suggestion

            # compressed and bitmask tensors
            # are no longer needed after decompression
            del layer.compressed
            del layer.bitmask
```